cmd/admin-handlers-site-replication.go

```diff
@@ -543,3 +546,44 @@ func (a adminAPIHandlers) SiteReplicationResyncOp(w http.ResponseWriter, r *http
 	}
 	writeSuccessResponseJSON(w, body)
 }
+
+// SiteReplicationDevNull - everything goes to io.Discard
+// [POST] /minio/admin/v3/site-replication/devnull
+func (a *adminAPIHandlers) SiteReplicationDevNull(w http.ResponseWriter, r *http.Request) {
+	globalSiteNetPerfRX.Connect()
+	defer globalSiteNetPerfRX.Disconnect()
+
+	connectTime := time.Now()
+	ctx := newContext(r, w, "SiteReplicationDevNull")
+	for {
+		n, err := io.CopyN(io.Discard, r.Body, 128*humanize.KiByte)
+		atomic.AddUint64(&globalSiteNetPerfRX.RX, uint64(n))
+		if err != nil && err != io.EOF {
```

Comment:
```suggestion
		if err != nil && err != io.EOF && err != io.ErrUnexpectedEOF{
```

no need to log unexpected EOF
```
Error: unexpected EOF (*errors.errorString)
       3: /Users/kp/code/src/github.com/minio/minio/internal/logger/logger.go:258:logger.LogIf()
       2: /Users/kp/code/src/github.com/minio/minio/cmd/admin-handlers-site-replication.go:565:cmd.(*adminAPIHandlers).SiteReplicationDevNull()
       1: net/http/server.go:2109:http.HandlerFunc.ServeHTTP()
```